<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>431</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget:pane {
border: 1px solid gray;
}
QTabWidget::pane {
  border: 1px solid lightgray;
  top:-1px; 
  background:rgb(0, 170, 255); 
} 

QTabBar::tab {
  background:rgb(255, 0, 0); 
  border: 1px rgb(0, 85, 255); 
  padding: 15px;
} 

QTabBar::tab:selected { 
  background: rgb(245, 245, 245); 
  margin-bottom: -1px; 
}
QTabWidget {  
    background-color:#0f2033;
    border-top: 2px solid #C2C7CB;  
} 
QTabWidget::pane { /* The tab widget frame */  
    border-top: 3px solid #C2C7CB;  
} 
 /* Style the tab using the tab sub-control. Note that 
    it reads QTabBar _not_ QTabWidget */  
QTabBar::tab {  
    background-color:#0f2033;
    border: 1px solid #4973b4;  
    min-width: 100px; 
    min-height:22px;
    padding: 0px  2px; 
    color: rgb(255, 255, 255);
    margin-left: 1px;
    font: 75 12pt &quot;NSimSun&quot;;
}  
  
QTabBar::tab:selected, QTabBar::tab:hover {  
    background-color:rgb(0, 0, 255);
    margin-top: 0px;
    
}  
  

  
QTabBar::tab:!selected {  
    margin-top: 5px; /* make non-selected tabs look smaller */  
}  

</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>ajouter</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>numero</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>140</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>200</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>destination</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_num">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nom">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_prenom">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_dest">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>280</y>
        <width>124</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: rgb(245, 245, 245);
background-color: rgb(0, 0, 255);
border-width: 1px;
border-color: rgb(0, 0, 255);
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 75 ;
padding-left: 1px;
padding-right: 1px;
min-width: 120px;
max-width: 50px;
min-height: 15px;
max-height: 15px;
font: 75 12pt &quot;NSimSun&quot;;
}</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>supprimer</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>Le numero du voyageur</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_numS">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>240</y>
        <width>124</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: rgb(245, 245, 245);
background-color: rgb(0, 0, 255);
border-width: 1px;
border-color: rgb(0, 0, 255);
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 75 ;
padding-left: 1px;
padding-right: 1px;
min-width: 120px;
max-width: 50px;
min-height: 15px;
max-height: 15px;
font: 75 12pt &quot;NSimSun&quot;;
}</string>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>modifier</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>num</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>280</y>
        <width>124</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: rgb(245, 245, 245);
background-color: rgb(0, 0, 255);
border-width: 1px;
border-color: rgb(0, 0, 255);
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 75 ;
padding-left: 1px;
padding-right: 1px;
min-width: 120px;
max-width: 50px;
min-height: 15px;
max-height: 15px;
font: 75 12pt &quot;NSimSun&quot;;
}</string>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nom_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_prenom_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_dest_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>220</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_num_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>destination</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>prenom</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>afficher</string>
     </attribute>
     <widget class="QTableView" name="tab_voyageur">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>431</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>966</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
